src/masternode-sync.cpp

```diff
@@ -508,6 +508,21 @@ void CMasternodeSync::ProcessTick()
     ReleaseNodes(vNodesCopy);
 }
 
+void CMasternodeSync::SendGovernanceSyncRequest(CNode* pnode, const char* strMessage, uint256 nHash)
```

Comment:
This function params are excessive imo: `strMessage` is always `NetMsgType::MNGOVERNANCESYNC` and `nHash` is always `uint256()`.